.github/.OwlBot.lock.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+docker:
```

Comment:
What's the maintenance regime for this and .OwlBot.yaml?

Reply:
Zero maintenance for these two files in .github.  When a new version of the post processor docker image is published, _and_ the new docker image produces different output than the previous docker image, Owl Bot will create a pull request that updates the sha in this lock file.

Reply:
Excellent - thanks for the info.